Exercises-4-OOP/src/inheritanceanimal/Cat.java

```diff
@@ -0,0 +1,29 @@
+package inheritanceanimal;
+
+public class Cat extends Mammal {
+
+	 String greets = "meow";
+
+	public Cat() {
+
+	}
+
+	public Cat(Mammal mammal1, String greets) {
+		super(mammal1.getName());
+		this.greets = greets;
+	}
+
+	public String getGreets() {
```

Comment:
```suggestion
	public String greets() {
```

The same for the `Dog` class.